frigate/object_detection.py

```diff
@@ -90,23 +202,26 @@ def run(self):
             with self.frame_lock:
                 cropped_frame = self.shared_frame[self.region_y_offset:self.region_y_offset+self.region_size, self.region_x_offset:self.region_x_offset+self.region_size].copy()
                 frame_time = self.frame_time.value
-            
-            # Resize to 300x300 if needed
-            if cropped_frame.shape != (300, 300, 3):
-                cropped_frame = cv2.resize(cropped_frame, dsize=(300, 300), interpolation=cv2.INTER_LINEAR)
-            # Expand dimensions since the model expects images to have shape: [1, 300, 300, 3]
-            frame_expanded = np.expand_dims(cropped_frame, axis=0)
 
+#            # Resize to 300x300 if needed
+#            if cropped_frame.shape != (300, 300, 3):
+#                cropped_frame = cv2.resize(cropped_frame, dsize=(300, 300), interpolation=cv2.INTER_LINEAR)
+#           # Expand dimensions since the model expects images to have shape: [1, 300, 300, 3]
+#           frame_expanded = np.expand_dims(cropped_frame, axis=0)
+
+            img = Image.fromarray(cropped_frame)
```

Comment:
What is happening here is the numpy array is converted to a PIL image, and then tensorflow converts it back to a 300x300 numpy array before detection. All that just adds processing delays.